HospitalWebApp/Controllers/MedicationController.cs

```diff
@@ -27,7 +27,7 @@ public IActionResult GetAllMedication()
         }
 
         [HttpGet]
-        [Route("find/{name}")]
+        [Route("{name}")]
```

Comment:
This can be problematic. In case where you, for some reason, change id of doctor to be string. REST will not know if you want to search doctor by id or name.

Reply:
Fixed.